comps/llms/text-generation/vllm-openvino/build_vllm_openvino.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+git clone --branch openvino-model-executor https://github.com/ilya-lavrenov/vllm.git
+cd ./vllm/
+docker build -t vllm:openvino -f Dockerfile.openvino .
```

Comment:
There are some instructions for adding `-e` for `docker run` above, so I also suggest that there should be a `--build-arg HTTP_PROXY=http://<DOMAIN-OR-IP>:PORT`, etc for `docker build`. Because I notice that many things need to be downloaded in [Dockerfile.openvino](https://github.com/ilya-lavrenov/vllm/blob/openvino-model-executor/Dockerfile.openvino)